tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "sourceMap": true,
         "rootDir": ".",
         "noImplicitReturns": true,
-        "strictNullChecks": true
+        "strictNullChecks": false
```

Comment:
I don't think it can be done without tweaking the code at least a little bit. You could try adding the postfix-`!` operator in places where the code fails to compile to assert the non-nullness of values.